Ports/zstd/patches/fix_soname_flags.patch

```diff
@@ -0,0 +1,11 @@
+--- zstd-1.5.0/lib/Makefile.orig	2021-11-01 15:49:13.000000000 +0100
++++ zstd-1.5.0/lib/Makefile	2021-11-01 15:49:16.000000000 +0100
+@@ -210,7 +210,7 @@
+   SHARED_EXT = dylib
```

Comment:
Although I can't spot anything with dylib ending up in the final file system, I feel like we should just remove the whole Darwin-specific case for clarity's sake, and just rely on what the library would do on a non-Apple system.

Reply:
That sounds like it makes more sense than what I did. Do you know how to test this?

Reply:
I think the only way that you can really test this is by building this on macOS (which I just realized I didn't, so of course I haven't seen any dylib files) and manually checking for *.dylib files in the resulting disk image.

Reply:
I took your suggestion, so it will now produce a .so library.